Lib/_pyrepl/windows_console.py

```diff
@@ -121,9 +128,8 @@ class _error(Exception):
 
 def _supports_vt():
     try:
-        import nt
-        return nt._supports_virtual_terminal()
-    except (ImportError, AttributeError):
+        nt._supports_virtual_terminal()
```

Comment:
```suggestion
        return nt._supports_virtual_terminal()
```
This is missing a return and would never enable virtual terminal anymore.